assets/express/src/api/healthz/routes/live.ts

```diff
@@ -1,4 +1,4 @@
-import { RouteOptions } from 'src/api/intefaces';
+import { RouteOptions } from 'src/api/interfaces';
```

Comment:
Should we have it like this or use `../../interfaces`, what do you think?

Reply:
This PR doesn't aim for fix imports